src/main/java/edu/ie3/datamodel/models/input/system/type/EvTypeInput.java

```diff
@@ -31,6 +33,7 @@ public class EvTypeInput extends SystemParticipantTypeInput {
    * @param eCons Consumed electric energy per driven distance
    * @param sRated Rated apparent power for this type of EV (typically in kW)
    * @param cosphiRated Power factor for this type of EV
+   * @param sRatedDC Rated apparent power for DC charging for this type of EV (typically in kW)
    */
```

Comment:
In my opinion, we should write ``active power`` or ``real power`` here, because we expect a value in ``kW``.

Reply:
For DC i would just say 'power'.